src/main/kotlin/com/exactpro/th2/ws/client/Main.kt

```diff
@@ -122,10 +124,21 @@ fun run(
     val outgoingSequence = createSequence()
 
     //TODO: add batching (by size or time)
-    val onMessage = { message: ByteArray, _: Boolean, direction: Direction ->
+    val onMessage = { message: ByteArray, textual: Boolean, direction: Direction, eventID: EventID? ->
+        eventID?.let {
```

Comment:
I'm not sure that it's a great idea to store an event per every send/received message. It could severely cripple performance in case of high load

Reply:
it's a valid point. Suggest to store only outgoing messages, since our priority is to find out whether it was sent or not.

Reply:
I doubt that we should store events for sent messages too

Reply:
Bump